### PR TITLE
[example/next-js-ondemand-isr] add example for revalidation of ISR pages using webhook

### PR DESCRIPTION
**Pull request by teleaziz**

### CODE REVIEW

File: examples/next-js-on-demand-isr/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+# Next.js + Builder.io On-Demand ISR
+Example of generating pages with Builder using nextjs ISR with [on-demand revalidation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#on-demand-revalidation)
```
Comment (ca136) [inline examples/next-js-on-demand-isr/README.md:2]:
```suggestion
Example of generating pages with Builder using Next.js ISR with [on-demand revalidation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#on-demand-revalidation).
```

File: examples/next-js-on-demand-isr/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+## Prerequisites
+
+Before using this example, make sure you have the following:
+
+- A [Builder.io](builder.io) account. Check out the [plans](https://www.builder.io/m/pricing), which range from our free tier to custom.
```
Comment (ca136) [inline examples/next-js-on-demand-isr/README.md:8]:
```suggestion
- A [Builder.io](https://builder.io) account. Check out the [plans](https://www.builder.io/m/pricing), which range from our free tier to custom.
```

File: examples/next-js-on-demand-isr/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+   ```shell
+   cd builder/examples/next-js-on-demand-isr
+   ```
+
+1. Open this directory in your favorite code editor.
+   You'll come back here when you're ready to connect your application to your Builder space.
+
+### Generating your Builder.io space
+
+If you've just created your Builder.io account and logged in for the first time, Builder prompts you to create a space with a new Builder site or add Builder to an existing application.
```
Comment (ca136) [inline examples/next-js-on-demand-isr/README.md:49]: Not sure this is relevant anymore since we deprecated hosting.

**Review by ca136 — APPROVED** [review]

File: examples/next-js-on-demand-isr/README.md (hunk 3 above)
Comment (teleaziz) [reply examples/next-js-on-demand-isr/README.md:49]: true, this whole section feels extraneous, i'l remove

**Pull request merged**